### ISSUE TITLE
Highcharts.setOptions with animation false is not working on V8.2.2, it was working fine on V4.2.1

### ISSUE THREAD

**Issue by AkibMankad** [highcharts/highcharts#15978]
Highcharts.setOptions({
	plotOptions: {
            series: {
                animation: false
            }
	}
});

#### Expected behaviour
It should disable the animation for all the charts

#### Actual behaviour
animation is not disabled

#### Product version
Highcharts version number --> V8.2.2

**Comment by raf18seb** [comment]
Hi @AkibMankad,

I am not able to reproduce the issue. It is working fine in v8.2.2 and in the newest v9.1.2, see demo: https://jsfiddle.net/BlackLabel/81mbdhkp/

You can contact our support so we will help you with finding a reason why it's not working for you.
https://www.highcharts.com/blog/support/

**Comment by AkibMankad** [comment]
Hello @raf18seb , 

Thanks for the quick check but, the issue here is initial loading has the animation true same as container3 example, but after that we are trying to disable the animation for the charts.

The reason to do this is, we have multiple views and by switching between the views, we don't want the animation. only on load of the page it required. 


try the same thing with  container3  in your fiddle,  it will not work.
[https://jsfiddle.net/BlackLabel/81mbdhkp/](url)

**Comment by raf18seb** [comment]
This is because the specific local chart's options have a higher priority than setting the global options through `setOptions`. This is intended behavior.

If you want to disable animation in all of your charts, you need to loop them and update options in every chart like this:
```js
setTimeout(function() {
  Highcharts.charts.forEach(function(chart) {
    chart.update({
      plotOptions: {
        series: {
          animation: false
        }
      }
    });
  });
}, 1100)
```

Demo: https://jsfiddle.net/BlackLabel/jtomx75n/

**Comment by AkibMankad** [comment]
Thank @raf18seb.

I got the solution for this, but, can you please confirm,  This intended behavior changed in which version? because, it was working fine on V4.2.2

**Comment by raf18seb** [comment]
@AkibMankad it's working the same in v4.2.2, see demo: https://jsfiddle.net/BlackLabel/24yxmn6r/

**Comment by AkibMankad** [comment]
@raf18seb ,

I have put exact issue we have described with animation in below two fiddle, could you please take a look.

V4.2.2 --> https://jsfiddle.net/e5Lgx8nm/
V8.2.2 --> https://jsfiddle.net/sdcbqn50/

**Comment by AkibMankad** [comment]
`hhh


`

**Comment by raf18seb** [comment]
The old behavior was a bug. It's been fixed since v7.1.2, see the [changelog](https://www.highcharts.com/blog/changelog/#highcharts-v7.1.2) and [related ticket](https://github.com/highcharts/highcharts/issues/3881)

**Comment by AkibMankad** [comment]
Thanks @raf18seb for the clarification and all the insight on this topic. 
Have a great day
